src/Handler/StreamHandler.php

```diff
@@ -77,7 +80,7 @@ private function invokeStats(
         RequestInterface $request,
         $startTime,
```

Comment:
missing type?

Reply:
I believe we can't be sure what this line returns

https://github.com/guzzle/guzzle/blob/master/src/Handler/StreamHandler.php#L36

So I left startTime as `mixed`.. other than that it is done

Reply:
I just noticed that on my previous commit I typehinted a different function from the one you commented. Will typehint this one too..

Reply:
OK.. added typehints now to the correct function :)

There are two functions with an argument `$startTime`... both added a typehint `mixed` since the line I mention above returns `float|mixed`